articles/flows/concepts/mobile-login-flow.md

```diff
@@ -14,7 +14,7 @@ useCase:
 ---
 # Mobile Login Flow
 
-During authentication, mobile/native applications require additional security as they:
+During authentication, mobile/native applications can use the OAuth 2.0 Authorization Code Flow, but they require additional security because they:
 
 * cannot securely store a Client Secret
 * may make use of a custom URL scheme to capture redirects (e.g., MyApp://) potentially allowing malicious applications to receive an Authorization Code from your Authorization Server.
```

Comment:
To mitigate this, OAuth 2.0 provides a version of the Authorization Code Flow ~which makes use of~ that uses a Proof Key for Code Exchange (PKCE) (defined in [OAuth 2.0 RFC 7636](https://tools.ietf.org/html/rfc7636)).